client/blocks/reader-site-notification-settings/index.jsx

```diff
@@ -154,13 +155,18 @@ class ReaderSiteNotificationSettings extends Component {
 					className="reader-site-notification-settings__popout"
 				>
 					<div className="reader-site-notification-settings__popout-toggle">
-						{ translate( 'Notify me of new posts' ) }
-						<Gridicon icon="bell" size={ 18 } />
-						<FormToggle
+						<CompactFormToggle
 							onChange={ this.toggleNewPostNotification }
 							checked={ sendNewPostsByNotification }
 							wrapperClassName="reader-site-notification-settings__popout-form-toggle"
+							id="reader-site-notification-settings__notifications"
```

Comment:
Do we only have one of these on the page?

Reply:
It appears so, but I think I've figured out a way to do this without adding `FormLabel`s manually. If I wrap the label text with `CompactFormToggle` it will show the component's children as the label. Trying that now...